tests/suites/test_suite_entropy.function

```diff
@@ -166,11 +166,10 @@ void entropy_init_free(int reinit)
 void entropy_seed_file(char *path, int ret)
 {
     mbedtls_entropy_context ctx;
+    mbedtls_entropy_init(&ctx);
```

Comment:
Note: this is why I'd like all contexts to have an initializer that can be used when declaring the variable, like PSA does. If even we keep getting it wrong, it means our API isn't as simple to use right as we want it to be.